src/main/java/com/example/gsheets/service/SheetService.java

```diff
@@ -1,8 +1,30 @@
 package com.example.gsheets.service;
 
+import com.example.gsheets.service.domain.Student;
 import org.springframework.stereotype.Service;
 
-@Service
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+
+import static com.example.gsheets.client.GoogleCredentialAPI.*;
+
+import static com.example.gsheets.client.GoogleCredentialAPI.getById;
 
+
+@Service
```

Comment:
why are you using @Service if you call it with operator "new"

Reply:
rewrite with beans. It fix this problem?

Reply:
done